docs/data-management.rst

```diff
@@ -292,6 +292,8 @@ annotations belong to others or in case these annotations belong
 to you but are also linked to some other objects
 in the original group which are not being moved.
 
+If you want to retain a copy of your data in it's current position, then you should :ref:`Duplicate<Duplicatecli>` it first. The :ref:`Duplication<Duplicatecli>` also bypasses the unlinked annotations problem highlighted in the previous paragraph. Because the :ref:`Duplicate feature<Duplicatecli>` creates new objects, which are not dependent on the originals, the relationships between these duplicates are then preserved even during a subsequent move to another group.
```

Comment:
```suggestion
If you want to retain a copy of your data in its current position, then you should :ref:`Duplicate<Duplicatecli>` it first. The :ref:`Duplication<Duplicatecli>` also bypasses the unlinked annotations problem highlighted in the previous paragraph. Because the :ref:`Duplicate feature<Duplicatecli>` creates new objects, which are not dependent on the originals, the relationships between these duplicates are then preserved even during a subsequent move to another group.
```

Reply:
More new lines would be good too ;)

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/19/commits/495e570239fb162473ae76ba229cc73c322dda73